dot/network/utils_test.go

```diff
@@ -20,31 +20,13 @@ func availablePort(t *testing.T) uint16 {
 
 	l, err := net.ListenTCP("tcp", addr)
 	require.NoError(t, err)
-	defer l.Close()
 
-	port := l.Addr().(*net.TCPAddr).Port
-	return uint16(port)
-}
-
-const portsAmount = 7100
-
-type portsQueue chan int
-
-func (p portsQueue) get() int {
-	return <-p
-}
-
-func (p portsQueue) put(port int) {
-	p <- port
-}
+	port := uint16(l.Addr().(*net.TCPAddr).Port)
 
-var availablePorts portsQueue
+	err = l.Close()
```

Comment:
A bit out of scope perhaps but this won't work to reserve a certain port for a certain test since it releases it back to the OS.

Reply:
yeah, I will fix it here -> #2105